File: hello.java
class JavaHello
{
    public static String[] planets = {"Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune", "Pluto"};

    public static void main(String args[])
    {
        int x = 0;
        while(x < 9){
            System.out.println(get_planet(x));
            x++;
        }
    }

    public static String get_planet(int index){
        return planets[index];
    }
}

File: hello.c
#include<stdio.h>

char * get_planet(int);
char * planets[9] = {"Mercury", "Venus", "Earth", "Mars", "Jupiter",
"Saturn", "Uranus", "Neptune", "Pluto"};

int main(){
    for(int i = 0; i < 10; i++){
        printf("%s", get_planet(i));
        printf("\n");
    }
    return 0;
}

char * get_planet(int index){
    if(index >= 0 && index <= 8){
        return planets[index];
    }
    else{
        return "I HATE YOU!!";
    }
}